src/main/java/net/sourceforge/jwbf/core/actions/HttpActionClient.java

```diff
@@ -74,434 +74,457 @@
  */
 public class HttpActionClient {
```

Comment:
I would prefer to rollback the change "Multimap to Map" because there will be many test failures, if the compile error (pluginManagement) is fixed. I would offer to change the this part later okay?